openshift/setup-teardown/params/COMMON_SETTINGS/api/api.config

```diff
@@ -7,6 +8,9 @@ API_BC_NODEJS_TEMPLATE_FILENAME=eagle-api.bc.json
 API_BC_NODEJS_PARAMS=api-1-nodejs-bc.params
 API_DC_MINIO_TEMPLATE_FOLDER_URL=https://raw.githubusercontent.com/your-github-org/your-api-repo/develop/openshift/templates/
 API_DC_MINIO_TEMPLATE_FILENAME=api-minio.dc.yaml
+API_BC_MINIO_TEMPLATE_FOLDER_URL=https://raw.githubusercontent.com/your-github-org/your-api-repo/develop/openshift/templates/
+API_BC_MINIO_TEMPLATE_FILENAME=minio-bc.json
```

Comment:
let's put this before the dc just so it flows better for the reader

Reply:
Done